packages/ui-utils/src/global.css

```
@@ -19,6 +19,7 @@
 @import './theme/tokens/space.css';
 
 @import './theme/typography/variables.css';
+@import './theme/craft.css';
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)